### ISSUE TITLE
No warning or error for bad target URL

### ISSUE THREAD

**Issue by aconfee** [shoreditch-ops/artillery#450]
This issue is a result of me being dumb, but I think it's worthwhile because it snagged me a number of times, and I may not be alone.

I accidentally added a target: localhost:3000, but forgot to include http:// -- http://localhost:3000. I received no warnings or errors for the incorrect URL. Since I'm switching between a number of different environments for setting up and testing, this has cost me 2 minutes here and there, and is just an annoyance that seems like it could be a quick win to fix. 

Thanks for the awesome work!

**Comment by hassy** [comment]
Yup, agreed @aconfee, something I've experienced myself a few times.
